plugins/finder

```diff
@@ -112,6 +115,36 @@ desktop(){
     killall Finder
 }
 
+path(){
+    case $1 in
+        [yY][eE][sS])
+            echo "Show folder path: YES"
+            defaults write com.apple.finder _FXShowPosixPathInTitle -bool true
+            ;;
+        [nN][oO])
+            echo "Show folder path: NO"
+            defaults write com.apple.finder _FXShowPosixPathInTitle -bool false
+            ;;
+        *)
+            EXTENSION_STATUS=$(defaults read NSGlobalDomain _FXShowPosixPathInTitle 2>/dev/null)
```

Comment:
I am getting an error here in macOS Sierra Version 10.12.4 (16E195)

```
$ m-cli finder showpath
We can't read _FXShowPosixPathInTitle property
```

I got it working applying the following patch:
```
- EXTENSION_STATUS=$(defaults read NSGlobalDomain _FXShowPosixPathInTitle 2>/dev/null)
+ EXTENSION_STATUS=$(defaults read com.apple.finder _FXShowPosixPathInTitle 2>/dev/null)
```